subsys/usb/usb_device.c

```diff
@@ -286,8 +284,6 @@ static void usb_handle_control_transfer(u8_t ep,
 	u32_t chunk = 0U;
 	struct usb_setup_packet *setup = &usb_dev.setup;
 
-	LOG_DBG("ep %x, status %x", ep, ep_status);
```

Comment:
Please do not remove this line, I have used it often to debug requests. Better change it to `LOG_DBG("ep 0x%02x, status 0x%02x", ep, ep_status);`

Reply:
ok